test/sagemaker_tests/pytorch/training/resources/fastai/cifar/cifar.py

```diff
@@ -1,46 +0,0 @@
-# Copyright 2018-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
why are these files removed?

Reply:
This is an old fastai test with deprecated code. Added the mnist distributed training test from upstream that test the same functionality.